networkx/algorithms/approximation/steinertree.py

```diff
@@ -0,0 +1,76 @@
+import networkx as nx
+from itertools import combinations, chain
+from networkx.utils import pairwise, not_implemented_for
+
+__all__ = ['metric_closure', 'steiner_tree']
+
+
+@not_implemented_for('directed')
+def metric_closure(G, nodes, weight="weight"):
+    """  Returns the sub-graph of the metric closure of the graph G induced by
+        the nodes.
+
+    Parameters:
```

Comment:
The heading `Parameters`, `Returns`, and `Notes` are all special section heading names specified by the [numpydoc][1] docstring format that we follow. There should be no trailing colon, and the number of dashes forming the underline must match the number of characters in the heading exactly.

The same comments apply in the other function below.

[1]: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt